src/main/java/org/breedinginsight/brapi/v2/services/BrAPIListService.java

```diff
@@ -72,19 +72,20 @@ public List<BrAPIListSummary> getListSummariesByTypeAndXref(
             // set the owner of the list items as the list owner
             BrAPIListsSingleResponse listDetails = listDAO.getListById(list.getListDbId(), program.getId());
             List<String> listItemNames = listDetails.getResult().getData();
-            switch (type) {
-                case OBSERVATIONVARIABLES:
-                    break;
-                case GERMPLASM:
-                default:
-                    String createdBy = germplasmDAO.getGermplasmByRawName(listItemNames, program.getId()).get(0)
-                            .getAdditionalInfo()
-                            .getAsJsonObject("createdBy")
-                            .get("userName")
-                            .getAsString();
-                    list.setListOwnerName(createdBy);
+            if (type != null) {
+                switch (type) {
+                    case OBSERVATIONVARIABLES:
+                        break;
+                    case GERMPLASM:
+                    default:
+                        String createdBy = germplasmDAO.getGermplasmByRawName(listItemNames, program.getId()).get(0)
+                                .getAdditionalInfo()
+                                .getAsJsonObject("createdBy")
+                                .get("userName")
+                                .getAsString();
+                        list.setListOwnerName(createdBy);
+                }
```

Comment:
I know this wasn't a change you made, but this code should probably be moved inside the `GERMPLASM` case

Reply:
Done, d029f0d6.

A switch doesn't really make sense at the moment, but I assume it will get more use in the future.